pxtsim/runtime.ts

```diff
@@ -435,6 +435,29 @@ namespace pxsim {
         return pxtcore.mkAction(0, s => _leave(s, f(s)))
     }
 
+    export class TimeoutScheduled {
+        constructor(id: number, fn: any, totalRuntime: number, timestampCall: number) {
```

Comment:
TypeScript has shortcuts for this kind of class; you can rewrite it like this:

```typescript
export class TimeoutScheduled {
    constructor(public id: number, public fn: Function, public totalRuntime: number, public timestampCall: number) { }
}
```

Same for `PausedTimeout`